src/Services/Manager.vala

```diff
@@ -233,4 +254,18 @@ public class BluetoothIndicator.Services.ObjectManager : Object {
     public static bool compare_devices (Device device, Device other) {
         return device.modalias == other.modalias;
     }
+
+    private async void setup_dbus_rfkill () {
+        try {
+            killer = yield Bus.get_proxy (
+                BusType.SESSION,
+                "org.gnome.SettingsDaemon.Rfkill",
+                "/org/gnome/SettingsDaemon/Rfkill"
+            );
+        } catch (Error e) {
+            killer = null;
+            return;
```

Comment:
This `return` seems unnecessary - the function ends after this point anyway.   We would normally put a `debug` or `warning` message here if an error is unexpected.